src/lug/System/Win32/PathImplWin32.cpp

```diff
@@ -0,0 +1,55 @@
+#include <lug/System/Win32/PathImplWin32.hpp>
+
+std::string lug::System::priv::PathImpl::getRoot() const {
+    char *homeDrive = nullptr;
+    homeDrive = getEnv("HOMEDRIVE");
+
+    if (homeDrive == nullptr) {
+        return std::string("");
+    }
+
+    std::string rootString;
+    rootString += homeDrive;
+    return rootString;
+}
+
+std::string lug::System::priv::PathImpl::getHome() const {
+    char *homeDrive = nullptr;
+    homeDrive = getEnv("HOMEDRIVE");
+    char *homePath = nullptr;
+    homePath = getEnv("HOMEPATH");
+
+    if (homeDrive == nullptr || homePath == nullptr) {
+        return std::string("");
+    }
+
+    std::string homeString;
+    homeString += homeDrive;

```

Comment:
Can you add a comment on why you need both `HOMEDRIVE` or `HOMEPATH`? The code is not clear enough, it seems only one is returned, but sometimes the other?